Source/JavaScriptCore/Configurations/JavaScriptCore.xcconfig

```diff
@@ -48,7 +51,10 @@ WK_SUPPORTS_OTHER_TAPI_FLAGS_STATICLIBS_WATCHOS_SINCE_10 = YES;
 WK_SUPPORTS_OTHER_TAPI_FLAGS_STATICLIBS_xros = YES;
 WK_SUPPORTS_OTHER_TAPI_FLAGS_STATICLIBS_xrsimulator = YES;
 
-SUPPORTS_TEXT_BASED_API = $(WK_AND_$(TAPI_USE_SRCROOT)_$(WK_SUPPORTS_OTHER_TAPI_FLAGS_STATICLIBS));
+SUPPORTS_TEXT_BASED_API = $(SUPPORTS_TEXT_BASED_API_$(JSC_OVERRIDE_SUPPORTS_TEXT_BASED_API))
+SUPPORTS_TEXT_BASED_API_ = $(WK_AND_$(TAPI_USE_SRCROOT)_$(WK_SUPPORTS_OTHER_TAPI_FLAGS_STATICLIBS));
+SUPPORTS_TEXT_BASED_API_NO = NO;
+SUPPORTS_TEXT_BASED_API_YES = YES;
```

Comment:
There is also a TAPI_ENABLE_VERIFICATION_MODE setting, could you see if that works to remove or speed up the generation phase? I think we should continue declaring TAPI support and only tell it to skip the verification step. We had some problems with incremental builds when toggling this setting.